runbotics/runbotics-orchestrator-ui/src/src-app/views/tenants/TenantsListView/TenantsListView.tsx

```diff
@@ -0,0 +1,109 @@
+import { VFC, useState, useEffect } from 'react';
+
+import { useSearchParams } from 'next/navigation';
+import { useRouter } from 'next/router';
+import { Tenant } from 'runbotics-common';
+
+import useTenantSearch from '#src-app/hooks/useTenantSearch';
+import useTranslations from '#src-app/hooks/useTranslations';
+import { useSelector } from '#src-app/store';
+import { tenantsSelector } from '#src-app/store/slices/Tenants';
+
+import TenantsListEditDialog from './TenantsListEdit';
+import TenantsListTable from './TenantsListTable/TenantsListTable';
+import { StyledActionsContainer, StyledButton, StyledTextField } from './TenantsListView.styles';
+import CreateTenantDialog from '../CreateTenantDialog';
+import { ROWS_PER_PAGE, DefaultPageValue } from '../TenantsBrowseView/TenantsBrowseView.utils';
+
+const TenantsListView: VFC = () => {
+    const { translate } = useTranslations();
+    const router = useRouter();
+    const searchParams = useSearchParams();
+
+    const currentPage = parseInt(searchParams.get('page'));
+    const pageSizeFromUrl = parseInt(searchParams.get('pageSize'));
+    const [page, setPage] = useState(currentPage ? currentPage : DefaultPageValue.PAGE);
+    const [limit, setLimit] = useState(
+        pageSizeFromUrl && ROWS_PER_PAGE.includes(pageSizeFromUrl)
+            ? pageSizeFromUrl
+            : DefaultPageValue.PAGE_SIZE
+    );
+
+    const { allByPage } = useSelector(tenantsSelector);
+
+    const { search, handleSearch, refreshSearch } = useTenantSearch({ page, pageSize: limit });
+
+    const [tenantData, setTenantData] = useState<Tenant>();
+    const [isEditDialogVisible, setIsEditDialogVisible] = useState(false);
+    const [isCreateDialogVisible, setIsCreateDialogVisible] = useState(false);
+
+    const handleOpenEditDialog = (rowData) => {
+        setIsEditDialogVisible(true);
+        setTenantData(rowData);
+    };
+
+    const handleCloseEditDialog = () => {
+        setIsEditDialogVisible(false);
+    };
+
+    const handleOpenCreateDialog = () => {
+        setIsCreateDialogVisible(true);
+    };
+
+    const handleCloseCreateDialog = () => {
+        setIsCreateDialogVisible(false);
+    };
```

Comment:
Close/Create dialog is quite well described. "handle" keyword is not neccessary here - it well describes general actions